kyaml/go.mod

```diff
@@ -11,13 +11,14 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/sergi/go-diff v1.1.0
 	github.com/spf13/cobra v1.0.0
-	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/objx v0.2.0 // indirect
 	github.com/stretchr/testify v1.5.1
 	github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15
 	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.19.16
```

Comment:
These are not real dependencies of the package, but are used in the tests and examples. Unfortunately go modules doesn't distinguish them.
I known kustomize doesn't want dependencies on k8s repos. If this is not allowed, we will need to replace some of the examples and tests.